tests/dialects/test_memref.py

```diff
@@ -486,3 +488,20 @@ def test_memref_copy():
         match="Expected source and destination to have the same element type.",
     ):
         copy.verify()
+
+
+def test_get_strides():
+    strided = StridedLayoutAttr((12, 4, 1), 5)
```

Comment:
```suggestion
    strided = StridedLayoutAttr((24, 4, 1), 5)
```
This should use a non-identity layout so that the test at line 507 actually checks whether the strides come from `strided`